pkg/scheduler/factory/factory.go

```diff
@@ -153,8 +153,6 @@ type Configurator interface {
 	GetClient() clientset.Interface
 
 	// TODO(#80216): Remove these methods from the interface.
```

Comment:
Can TODO be deleted?

Reply:
Not yet. `GetScheduledPodLister` can be removed as well, but it requires more changes. Maybe `GetClient` too. I hape a WIP.